portlet/exoadmin/package.json

```diff
@@ -35,5 +35,8 @@
     "webpack-cli": "^3.1.1",
     "webpack-dev-server": "^3.11.2",
     "webpack-merge": "^4.1.3"
+  },
+  "dependencies": {
+    "axios": "^0.19.0"
```

Comment:
We should not use axios. We have to use Javascript Fetch API

Reply:
understood